grant-project/zcash-grant-system
# Update vulnerable packages

Closes #264.

### What This Does

* Updates all vulnerable packages
* Fixes react-mde behavior and styles due to major version updates required
    * This update should save ~400k off of some of our bundles that include it due to them dropping draft.js, awww yeaaaa

### Steps to Test

* Confirm the whole site more or less works
* Confirm comment editing and proposal editing behave more or less correctly
